web/src/components/troubleshoot/SupportBundleAnalysis.jsx

```diff
@@ -137,10 +139,10 @@ export class SupportBundleAnalysis extends React.Component {
   };
 
   getSupportBundle = async () => {
-    this.setState({
-      loading: true,
-      getSupportBundleErrMsg: "",
+    this.props.updateState({
       displayErrorModal: false,
+      getSupportBundleErrMsg: "",
+      loading: true,
```

Comment:
because pollForBundleAnalysisProgress is done in parent component, some of the states have to be lifted up to be shared across the component. 
Let me know if there's a better way for this.